easybuild/easyconfigs/g/geopy/geopy-2.3.0-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'PythonBundle'
+
+name = 'geopy'
+version = '2.3.0'
+
+homepage = 'https://geopy.readthedocs.io/en/stable/'
+description = """
+geopy is a Python client for several popular geocoding web services.
+geopy makes it easy for Python developers to locate the coordinates of addresses, cities, countries,
+and landmarks across the globe using third-party geocoders and other data sources.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+builddependencies = [
+    ('binutils', '2.38'),
+]
+
+dependencies = [('Python', '3.10.4')]
+
+use_pip = True
+
+exts_list = [
+    ('geographiclib', '1.50', {
```

Comment:
`geographiclib` is not the same as `geopy`.
which one is it that you need?

Reply:
@garadar please answer my question?

Reply:
I need geopy, but I don't understand your question very well, I pick this recipe from someone else,

What does exts_list?

Reply:
`ext_list` is the list of extensions, i.e. the python packages that will be installed.
if you need `geopy`, you should include it in this list as follows:
```python
    (name, version, {
        'checksums': ['892b219413e7955587b029949af3a1949c6fbac9d5ad17b79d850718f6a9550f'],
    }),
```

looking at easyconfig `geopy-2.1.0-GCCcore-10.2.0.eb` in the EB repo, `geographiclib` seems to be a dependency of `geopy`, so it can stay.